public/content/sv/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,55 @@
+# Välkommen till D
+
+Välkommen till den interaktiva rundturen av *Programmings språket D*.
+
+{{#dmanmobile}}
+
+Denna rundtur ger en överblick av detta __kraftfulla__ och __målande__
```

Comment:
`målande` for me doesn't really feel like the right word.
I think something like `uttrycksfulla` sounds better.